### PR TITLE
First version

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
We need a new service to handle events that are sent to Fieldwork, so that we can scale it for receipting, refusals etc.

# What has changed
Created new Fieldwork Adapter service, which listens to the JSON output message on the `Action.Field` queue from the Action Scheduler, and then republishes as XML to a queue called `RM.Field`.

# How to test?
Run acceptance tests with the new Fieldwork Adapter service running and the new version of Action Scheduler, both from branch `spike-fieldwork-adapter`.

# Links
Trello: https://trello.com/c/WE8EGeL4

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[SPIKE: Fieldwork Adapter Service (0.5 days)](https://trello.com/c/WE8EGeL4/970-spike-fieldwork-adapter-service-05-days)

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    <dependency>
+      <groupId>org.springframework.integration</groupId>
+      <artifactId>spring-integration-amqp</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.integration</groupId>
+      <artifactId>spring-integration-xml</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
```
Comment (LukeLoze1) [inline pom.xml]: is the web import required? tests pass without

Comment (NickGrantONS) [reply pom.xml]: I'll take it out

Comment (NickGrantONS) [reply pom.xml]: although it'll need to go back in if we call any REST APIs

File: src/main/java/uk/gov/ons/census/fwmtadapter/healthcheck/HeathCheck.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package uk.gov.ons.census.fwmtadapter.healthcheck;
```
Comment (davidmort) [inline src/main/java/uk/gov/ons/census/fwmtadapter/healthcheck/HeathCheck.java]: filename isn't spelled correctly, should be HealthCheck.java not HeathCheck.java

Comment (NickGrantONS) [reply src/main/java/uk/gov/ons/census/fwmtadapter/healthcheck/HeathCheck.java]: Fixed. Fixed yesterday actually but forgot to push.

**Pull request merged**